sources/T-Editing-Guidelines-201602.adoc

```diff
@@ -0,0 +1,843 @@
+* Definitions should appear in alphabetical order.
+
+
+[appendix]
+== URI naming scheme for the identification of XML namespaces defined in ITU-T Recommendations
+
+[[a_C_1Introduction]]
+=== Introduction
+
+ITU-T Recommendations increasingly define XML schemas. To answer the growing demand of universal resource identifiers (URIs) for referencing XML namespaces in ITU-T Recommendations, ITU reserved the following URL: +
+[align=center]
+http://www.itu.int/xml-namespace/
+
+This annex gives the rules for allocating URIs to XML namespaces defined in ITU-T Recommendations.
+
+
+[[a_C_2Proposed_structure_for_URIs_defined_in_ITU-T_Recommendations]]
+=== Proposed structure for URIs defined in ITU-T Recommendations
+
+In order to guarantee the uniqueness of namespaces defined in ITU-T Recommendations as well as to improve their visibility, any XML namespace will be referenced with a URI of the form: +
+[align=center]
+*http://www.itu.int/xml-namespace/itu-t/x.nnn.pp/name*
+
+where:
+
+"x":: is the ITU-T Recommendation series letter, in lower case, of the referenced Recommendation, i.e., "a" to "z"
+
+"nnn":: is the main number of the Recommendation
+
+"pp":: is the sub-number, if any
+
+"name":: is the namespace(s) defined in the Recommendation
+
+The study group responsible for the Recommendation identified by the namespace, "*http://www.itu.int/xml-namespace/itu-t/x.nnn.pp*/", has the authority to allocate one or more subsequent arcs. Moreover, that study group has the authority to insert a subsequent arc to handle version numbers or approval dates, if there is a need to differentiate among editions of that Recommendation. Such allocations shall be published in the Recommendation making those allocations. All arcs shall preferably be written in lower case.
+
+It is noted that this naming scheme can easily be extended to accommodate other types of documents produced in ITU.
+
+[[a_C_3Examples_of_use]]
+=== Examples of use
+
+[circle]
+* Recommendation ITU-T X.1206 (2008), _A vendor-neutral framework for automatic notification of security related information and dissemination of updates_. +
+[align=center]
+*http://www.itu.int/xml-namespace/itu-t/x.1206/core/*
+
+* Draft Recommendation ITU-T H.325, _Advanced multimedia systems for next generation and other packet-switched networks_. +
+[align=center]
+*http://www.itu.int/xml-namespace/itu-t/h.325/core/*
+
+
+ 
+[appendix]
+== Actions required to improve the quality of ITU-T Recommendations
+
+In an ITU-T meeting on a Question and before a draft Recommendation is proposed for approval, consent or determination, the rapporteur should ensure that all of the bullet points of the check list below have been reviewed and that they have been fulfilled adequately. This should also be reflected in the report of the Question.
+
+NOTE: Additional guidance is provided in clause 5 of <<b-ITU-T_Rap>>.
+
+The draft <title of the draft>, which is proposed for approval, consent or determination:
+
+[style=circle]
```

Comment:
I don't think we should be changing the list styles here, the priority is to keep things for their meaning, not their presentation. (which is the intention of ITU)

Reply:
Removing [style=circle]

Reply:
As it happens, there are styling commands which Metanorma completely ignores anyway, and choice of bullet is one of them. (And as we just saw with steps, the only way we will allow a bullet variant is if there is already a defined CSS style in the Word template -- meaning, as a CSS class of lists. We probably can apply Word CSS for bullet variants inline, on an ad hoc basis (parsing "style"); but that goes against our design philosophy of constraining rendering to semantically meaningful distinctions.